### PR TITLE
Update the interactive_flex example for the new event handlers

### PR DESCRIPTION
**Pull request by jason-simmons**

### CODE REVIEW

**Comment by abarth** [comment]
LGTM

File: examples/rendering/interactive_flex.dart [hunk 1]
```diff
@@ -33,18 +37,27 @@ class RenderImageGrow extends RenderImage {
...
+  if (eventType == "back") {
     activity.finishCurrentActivity();
   }
 }
 
+final Map<int, Touch> touches = <int, Touch>{};
+
+void handlePointerPacket(ByteData serializedPacket) {
+  bindings.Message message = new bindings.Message(
+      serializedPacket, <core.MojoHandle>[]);
```
Comment (Hixie) [inline examples/rendering/interactive_flex.dart]: 2-space indent, ) on the next line

File: examples/rendering/interactive_flex.dart [hunk 2]
```diff
@@ -33,18 +37,27 @@ class RenderImageGrow extends RenderImage {
...
+  PointerPacket packet = PointerPacket.deserialize(message);
+
+  for (Pointer pointer in packet.pointers) {
+    if (pointer.type == PointerType.MOVE) {
```
Comment (Hixie) [inline examples/rendering/interactive_flex.dart]: no braces around one-line blocks

**Comment by Hixie** [comment]
LGTM modulo nits. Thanks!

File: examples/rendering/interactive_flex.dart (hunk 1 above)
Comment (jason-simmons) [inline examples/rendering/interactive_flex.dart]: done

File: examples/rendering/interactive_flex.dart (hunk 2 above)
Comment (jason-simmons) [inline examples/rendering/interactive_flex.dart]: done

**Pull request merged**